docs/available_checks.md

```diff
@@ -66,6 +66,7 @@ This document is a list of tables with a description and link to documentation f
 | Check Name | Check Description |
 | :--------- | :---------------- |
 | BoundaryIntersectionCheck |  This check is designed to scan relations marked as boundaries or with ways marked as boundaries and flag them for intersections with other boundaries of the same type. |
+| [DuplicateRelationCheck](checks/DuplicateRelationCheck.md) | This check attempts to idenfy duplicate Relations base on the same OSM tags and the same members with same roles. |
```

Comment:
typo: This check attempts to ~idenfy~ identify duplicate Relations ~base~ based on the same OSM tags and the same members with same roles.

Reply:
thanks, I will be careful when typing, my keyboard is lagging.

Reply:
@sayas01, I made typo in one of my commits message, I tried to change it, but still show in git log, just wondering if I can close this pull request and make a new PR for this check?  so the commit message with typo will not be in git log.

Reply:
@sayas01, I tried to change one pushed commit description, but commits got messed up for that, new repository will be more clean for this check, so I closed this one, and open a new PR for this check.